src/components/common/ColonyActions/ActionsListItem/ActionsListItem.tsx

```diff
@@ -69,6 +76,27 @@ const ActionsListItem = ({
 
   const status = ListItemStatus.Defused;
 
+  const fromDomain = safeTransaction
+    ? initiatorColony?.domains?.items[0]
+    : itemDomain;
```

Comment:
According to my review about the colony address wrongfully being the initiator, you might want to update this so it _assumes_ that the Root domain is the `fromDomain` instead of getting it from `initiatorColony`